CMakeLists.txt

```
@@ -34,6 +34,10 @@ include(pybind11)
 include(GoogleTest)
 include(GoogleBenchmark)
 
+if("${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}" VERSION_LESS_EQUAL "3.4")
```

Would this be equivalent (and less code)?
```suggestion
if("${PYTHON_VERSION_STRING}" VERSION_LESS_EQUAL "3.4.0")
```

---

Also should the check be less, rather than less than?

---

If it works, happy to accept the changes.

---

3.4.0 doesn't work. In fact I'm not aware that we have tested. below 3.6. Maybe the check should be Less 3.6 is failure?

---

Actually this doesn't work. Not sure what is setting those variables but it does not seem to be [`FindPython`](https://cmake.org/cmake/help/v3.15/module/FindPython.html#module:FindPython).

The version requirements here (> 3.4) are different to the changes in the readme (>= 3.4).

---

It's PyBind11 not `FindPython`

---

A part of me had hoped pybind11 would have just used FindPython...

(they do use FindPythonInterp, which should still set `PYTHON_VERSION_STRING`)

---

Great. I'll make the changes you've proposed.

---

Actually, don't bother. `PYTHON_VERSION_STRING` is still not set...

---

Unfortunately this Cmake var is not propagated through, shame as it would have been a neater solution.